socorro/unittest/cron/crontabber_tests_base.py

```diff
@@ -0,0 +1,257 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# NOTE(willkg): This file is based on crontabber's crontabber/tests/base.py
+# file, then adjusted so it doesn't use nose.
```

Comment:
It's the whole file with three adjustments:

1. it no longer imports nose things
2. it no longer marks `IntegrationTestCaseBase` with the postgres nose attribute (helps to constrain which tests get run based on categories)
3. it no longer uses `eq_`

https://github.com/mozilla/crontabber/blob/master/crontabber/tests/base.py